airbyte-server/src/main/java/io/airbyte/server/apis/factories/AttemptApiFactory.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.server.apis.factories;
+
+import io.airbyte.persistence.job.JobPersistence;
+import io.airbyte.server.apis.AttemptApiImpl;
+import java.util.Map;
+import org.glassfish.hk2.api.Factory;
+import org.slf4j.MDC;
+
+public class AttemptApiFactory implements Factory<AttemptApiImpl> {
```

Comment:
Is this going to be replaced with a [Micronaut Factory](https://docs.micronaut.io/latest/api/io/micronaut/context/annotation/Factory.html) when this is all said-and-done?

Reply:
I don't know if this is compatible with the server we are using (glassfish). I'll need to see if I can convert it when moving to micronaut for the server.